### PR TITLE
q45

### PR DESCRIPTION
**Pull request by Navleenkaursandhu**
q45: generate pronounceable words
image

### CODE REVIEW

File: Javascript/q45.html [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+          }
+        }
+      }
+
+      return finalString;
+
+
+     
+
+      // TODO Create mutable string here which will be filled up with vowels and consonants
```
Comment (Mantaseus) [inline Javascript/q45.html:64]: Remove these comments

File: Javascript/q45.html [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+      </td>
+    </tr>
+    <tr>
+      <th>Count</th>
+      <td>
+        <input id="count-no" type="number"></input>
+      </td>
+    </tr>
+  </table>
+  <button onclick="string()">Generate random</button>
```
Comment (Mantaseus) [inline Javascript/q45.html:102]: What is `string()`? Use some proper function name like `generateString()` or something

**Review by Mantaseus — APPROVED** [review]

File: Javascript/q45.html (hunk 1 above)
Comment (Navleenkaursandhu) [reply Javascript/q45.html]: done

File: Javascript/q45.html (hunk 2 above)
Comment (Navleenkaursandhu) [reply Javascript/q45.html]: done

**Pull request merged**